addons/website_event/static/tests/tours/website_event_tour.js

```diff
@@ -0,0 +1,45 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add('Website_event_tour', {
```

Comment:
We already have a 'website_event_tour' tour ... and actually can't we add the step into 'test_tickets_questions' tour for example ? That way we don't have a new tour just for that (as a new tour takes quite a lot of time to execute), but it is tested. Could you check ?

Reply:
Hello @tde-banana-odoo 

Changes have been made 😊

Thanks!!